tests/tests.go

```diff
@@ -128,9 +128,9 @@ func addNetworksToPod(pod *corev1.Pod, networks []map[string]string) {
 	}
 }
 
-func findPodByUid(pods *corev1.PodList, podToFind corev1.Pod) *corev1.Pod {
```

Comment:
We can keep this change, Its more accurate when looking for a pod.

Reply:
yes, but let's insert it in a different PR, as it's not solely related to statefulSet, and I want to remove the commit as a whole